apps/guardian-ui/src/languages/de.json

```
@@ -176,8 +176,8 @@
     "submit": "Einreichen"
   },
   "role-selector": {
-    "disclaimer-title": "Fedimint ist Alpha-Software",
-    "disclaimer-text": "Sie verwenden eine sehr frühe Version von Fedimint. Bitte verwenden Sie sie mit Vorsicht und melden Sie Probleme den Fedimint-Entwicklern auf Github.",
+    "disclaimer-title": "Fedimint ist Beta-Software",
+    "disclaimer-text": "Bitte melden Sie Probleme unter https://github.com/fedimint/fedimint/issues",
```

```suggestion
    "disclaimer-title": "Fedimint ist Beta-Software",
    "disclaimer-text": "Bitte melde Probleme unter https://github.com/fedimint/fedimint/issues",
```

Much less formal, we shouldn't sound too corporate imo.

---

can update my system prompt for the translation script haha

```
async function translateWithOpenAI(text, targetLang) {
  const prompt = `Translate the following text to ${targetLang}. 
  Return only the translated string, without any additional text or explanations. 
  Be more informal and less corporate, use melde instead of melden Sie:\n\n${text}`;
  ```